### DISCUSSION TITLE
Suggestion: add `homing_positive_dir: false` to the sections [stepper_x/y/z]

### DISCUSSION THREAD

**Discussion by 1coderookie** [1coderookie/Klipper4KobraGoNeo#4] [General]
As the limit switches are on the minimum positions, I'd suggest to add `homing_positive_dir: false` to each of the three axes in the sections [stepper_x], [stepper_y], [stepper_z].   
This is the belonging description from the Klipper doc:  
```
#homing_positive_dir:
#   If true, homing will cause the stepper to move in a positive
#   direction (away from zero); if false, home towards zero. It is
#   better to use the default than to specify this parameter. The
#   default is true if position_endstop is near position_max and false
#   if near position_min.
```
@xiaopeng12138 maybe you want to add that to the cfg-file for the Go in the Klipper repo as well?

**Comment by xiaopeng12138** [comment]
I don't think this is necessary since the klipper handles this just fine. And according to https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md its better to avoid defining field values that are set to their default value.

**Reply by 1coderookie** [reply]
I just realized the sentence "default is ... false if near position_min" and that's the case here, dind't realize it before LOL  
Thx, I'll remove it from my cfg then I guess..

**Reply by xiaopeng12138** [reply]
wrong thread sry

**Reply by 1coderookie** [reply]
..no prob..
